hivemind/server/layers/common.py

```diff
@@ -1,13 +1,15 @@
 import torch
 from torch import nn as nn
 
+from hivemind.server.layers.custom_experts import register_expert_class
 
 # https://github.com/huggingface/transformers/blob/master/src/transformers/activations.py
 @torch.jit.script
 def gelu_fast(x):
     return 0.5 * x * (1.0 + torch.tanh(x * 0.7978845608 * (1.0 + 0.044715 * x * x)))
 
-
+ffn_sample_input = lambda batch_size, hid_dim: torch.empty((batch_size, hid_dim))
```

Comment:
Extra lines are missing

Reply:
reference: https://www.python.org/dev/peps/pep-0008/#blank-lines

Reply:
Added lines in both files